docs/userguide/transfer_learning.md

```diff
@@ -1,19 +1,59 @@
-offers only a single one.
+The term "Transfer Learning" is used in a lot of different ways. Within BayBE,
+"Transfer Learning" refers to combining data from multiple campaigns that are from 
+similar contexts. Depending on the field, this might also be known as "contextual
+learning". In BayBE we also refer to the different contexts as "tasks".
+```
+
+## When and When Not To Use Transfer Learning
+
+Thinking about combining data from different campaigns, a naive approach to achieve
+this is to quantify the differences between the contexts via one or few explicitly
+measured parameters. Examples where this is possible are:
+* Data for a chemical reaction that was optimized at a certain temperature can be used 
+  in a new campaign, where the same reaction should be optimized again at a different 
+  temperature.
+* Optimization of a simulation involving different particle sizes: The data obtained at
+  a smaller particle size can be utilized when starting a new optimization for a larger
+  particle size.
+
+However, it can be difficult to quantify the differences between the campaigns. This
+might be the case if the parameters distinguishing the contexts i) are not available and
+cannot be measured anymore; ii) are too many and explicitly modelling them is out of
+question; or iii) are simply unknown.
+
+Examples for situations where explicit quantification of the context can be difficult:
+* Cell culture media optimizations for different cell types: Cell types differ among 
+  many possible descriptors, and it is not known a priori which ones are relevant to a
+  newly started campaign.
+* Optimization in industrial black-box contexts: When materials (such as cell lines or
+  complex substances) stem from customers, they can come uncharacterized.
+* Transfer of a complicated process to another location. The transferred machinery will
+  likely require a new calibration / optimization and could benefit from the other
+  location's data. However, is not necessarily clear what parameters differentiate the
+  location context.
+
+**Transfer Learning** in BayBE offers something for examples such as the latter, because
+it abstracts the differences between campaigns into one dimension. Over the course of
+an ongoing campaign, the covariance between current campaign data and data from
+previous campaigns can be learned, basically enabling us to utilize previous data. For
+instance, this can unlock data treasures coming from similar but not identical campaigns
+accumulated over many years.
+
+```{admonition} Expectations
+:class: important
+Because of the need to learn the covariance between tasks, transfer learning is not a
+magic method for zero-shot learning. It will always need data from an ongoing campaign
+to understand how other campaigns' data are related. Overall, if correlated task data
+were provided, the optimization of new campaigns can experience a dramatic speedup.
```

Comment:
Would add a warning that learning can also be worse if you add stuff that is unrelated.

Reply:
added 
![image](https://github.com/emdgroup/baybe/assets/17951239/78cad5ff-c7ea-4b76-a0d6-46eadce33f13)